.pre-commit-config.yaml

```diff
@@ -0,0 +1,21 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+#
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.6.0
```

Comment:
Let's use the latest versions of this hook and `typos` below. I pushed a commit to your branch doing that.